docs/static/openapi.yml

```diff
@@ -54702,6 +54790,23 @@ definitions:
       epochBlockHash:
         type: string
         format: byte
+  lavanet.lava.pairing.Badge:
```

Comment:
🤩 

@orenl-lava FYI, because we alreaydy have breaking changes upcoming soon, we decided to introduce future changes to Badges. If by chance you feel ready to review this as part of your badge design...